clusterloader2/pkg/framework/client/objects.go

```diff
@@ -62,7 +62,7 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc) error {
 // IsRetryableAPIError verifies whether the error is retryable.
 func IsRetryableAPIError(err error) bool {
 	// These errors may indicate a transient error that we can retry in tests.
-	if apierrs.IsInternalError(err) || apierrs.IsTimeout(err) || apierrs.IsServerTimeout(err) ||
+	if apierrs.IsInternalError(err) || apierrs.IsTimeout(err) || apierrs.IsServerTimeout(err) || apierrs.IsUnauthorized(err)  ||
```

Comment:
In general, one doesn't retry 401 errors. Could you also add the explanation (from PR's description) why you do it here?

Reply:
I added comment.